.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
Personally, I like the ability to create blank issues if necessary. I would like to get a second opinion on this

Reply:
+

Reply:
Personally, I am one for standardisation to limit the initial back and forth when understanding why an issue has occurred. Also, for alignment between projects, I have created these issues following the format of the core [surrealdb](https://github.com/surrealdb/surrealdb/blob/d2571a6b290b5dee8ff8bca823c740ca1b752832/.github/ISSUE_TEMPLATE/config.yml#L1) project.

Happy either way, though 👍🏻